### DISCUSSION TITLE
Weird characters in vimtree

### DISCUSSION THREAD

**Discussion by janban-outlook** [NvChad/NvChad#2358] [Q&A] [answered]
I just installed everything, using Jetbrains Mono nerdfonts.
All fonts seems to render well, like the find icon and the icons in the right down corner, except for Vimtree. How can I solve this?

![image](https://github.com/NvChad/NvChad/assets/43573150/6bc27331-079b-4e7d-b426-5451df81c4bc)

**Comment by siduck** [comment, marked as answer]
set that font as main font in your terminal

**Reply by janban-outlook** [reply]
I did, but it didn't work for the treeview. Today I discovered that it works after I booted my laptop. Apparently it needs a reboot.
